salt/kubernetes-minion/stop.sls

```diff
@@ -0,0 +1,31 @@
+# Stop and disable the Kubernetes minion daemons, ensuring pods have been drained
+# and the node is marked as unschedulable.
+
+# If this, or the state below, fails we should ignore it and proceed anyway as
+# Kubernetes will recover.
+cordon-kubelet:
+  cmd.run:
+    - name: |
+        kubectl --kubeconfig /var/lib/kubelet/kubeconfig cordon {{ grains['id'] }}.{{ pillar['internal_infra_domain'] }}
```

Comment:
Already done in in #85 (Graham sucked this change in from there to make use of it)